testing/mock/mock.go

```diff
@@ -4,16 +4,17 @@ import (
 	"encoding/json"
 	"fmt"
 
+	"github.com/gorilla/mux"
+	"github.com/grpc-ecosystem/grpc-gateway/runtime"
+	"github.com/spf13/cobra"
+	abci "github.com/tendermint/tendermint/abci/types"
+
 	"github.com/cosmos/cosmos-sdk/client"
 	"github.com/cosmos/cosmos-sdk/codec"
 	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/cosmos-sdk/types/module"
 	capabilitytypes "github.com/cosmos/cosmos-sdk/x/capability/types"
```

Comment:
normally we would combine these sets of imports since they are both external

Reply:
checkout #1770

Reply:
on this now

Reply:
Ah, sorry. I see you fixed it for this file, but I intended to communicate that the imports have been changed in almost all the files in this pr. Is it possible to fix those files as well?

Reply:
sure!

Reply:
I read the other issue... trying to devise a way to do the desired fix

Reply:
https://github.com/incu6us/goimports-reviser